src/plugins/pluginUtils.ts

```diff
@@ -32,6 +33,9 @@ export function simplifiedRegExp(input: undefined | string | RegExp): RegExp {
         case '/':
           r += '(\\/|\\\\)';
           break;
+        case '\\':
```

Comment:
so that's when a absolute path on windows is passed right?

Reply:
do you think you can give a test? that's quite an important part of the process

Reply:
> so that's when a absolute path on windows is passed right?

All paths have `\\` while linux/max uses `/`.
Here is a typical path from my vmware, the `\` will ofc be escaped.. so really `\\`
`C:\Users\Developer\Desktop\New folder\plugins\core`
This is why I removed the split/join part and just used this.


> do you think you can give a test? that's quite an important part of the process

I will try and give this and sparky some tests.
Limted time this week and not very good at tests, so its just going to a little while :-)
But will try and improve 👍 
This is under plugin utils, but all path matching prob should be move to 1 place. so no-mather what you do we all use same tested functions.
Hopyfully Im the only one using npm scripts in subfolders and windows 😂